column/api/backend/cache/__init__.py

```diff
@@ -0,0 +1,21 @@
+import local_mem
```

Comment:
always import full path.  should be:

from column.api.backend.cache import local_mem